__tests__/unit/lib/utils/cliHelper.test.js

```diff
@@ -222,4 +224,42 @@ describe(`test if the application`, () => {
       format(messages.errorParameterIsNotGitSHA, 'from', TAG_REF_TYPE)
     )
   })
+
+  test('do not throw errors when repo contains submodule git file', async () => {
+    fs.errorMode = false
+    fs.statErrorMode = false
+    fs.__setMockFiles({
+      ...mockFiles,
+      'submodule/.git': 'lorem ipsum',
+    })
+    let cliHelper = new CLIHelper({
+      ...testConfig,
+      generateDelta: false,
+      repo: 'submodule/',
+    })
+    expect.assertions(1)
+    await expect(cliHelper.validateConfig()).rejects.not.toThrow(
+      format(messages.errorPathIsNotGit, 'submodule/')
+    )
+  })
+
+  test('do not throw errors when repo submodule git folder', async () => {
+    fs.errorMode = false
+    fs.statErrorMode = false
```

Comment:
**remark(_nit_):** I think it is not necessary here because done in the before all

Reply:
I thought so too, but it fails if I take it out. It took me a while to figure that out as it didn't make sense to me. I think it might have something to do with the parallelism on tests, but I'm not sure.